wp-content/plugins/core/src/Blocks/Types/Accordion/Accordion.php

```diff
@@ -3,78 +3,95 @@
 
 namespace Tribe\Project\Blocks\Types\Accordion;
 
-use Tribe\Gutenpanels\Blocks\Block_Type_Interface;
-use Tribe\Gutenpanels\Blocks\Sections\Content_Section;
-use Tribe\Gutenpanels\Blocks\Sections\Toolbar_Section;
-use Tribe\Project\Blocks\Block_Type_Config;
-use Tribe\Project\Blocks\Types\Accordion\Support\Accordion_Section;
+use Tribe\Libs\ACF\Field;
+use Tribe\Libs\ACF\Block;
+use Tribe\Libs\ACF\Block_Config;
+use Tribe\Libs\ACF\Repeater;
 
-class Accordion extends Block_Type_Config {
-	public const NAME = 'tribe/accordion';
+class Accordion extends Block_Config {
+	public const NAME = 'accordion';
 
-	public const TITLE       = 'title';
-	public const DESCRIPTION = 'description';
-	public const ACCORDION   = 'accordion';
+	public const TITLE             = 'title';
+	public const DESCRIPTION       = 'description';
+	public const ACCORDION         = 'accordion';
+	public const ACCORDION_HEADER  = 'accordion_header';
```

Comment:
I wonder if we should name these `row_header` & `row_content` or something similar? I feel like `accordion_header` etc is a bit confusing.